packagingrunner/xenial/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/dennybaa/droneruby:trusty-rbenv
+FROM quay.io/dennybaa/droneruby:xenial-rbenv
```

Comment:
This change is not needed and doesn't affect producing trusty packages.

```suggestion
FROM quay.io/dennybaa/droneruby:trusty-rbenv
```

Reply:
I agree